packages/components/src/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -0,0 +1,190 @@
+                onToggle={onToggle}
+            >
+                <SampleTypePropertiesPanelImpl {...this.props} />
+            </DomainPropertiesPanelProvider>
+        );
+    }
+}
+
+class SampleTypePropertiesPanelImpl extends React.Component<Props,State> {
+    static defaultProps = {
+        noun: 'Sample Type',
+        nameExpressionInfoUrl: '',
+        nameExpressionPlaceholder: 'S-\${now:date}-\${dailySampleCount}',
+        initCollapsed: false,
+        validate: false,
+        appPropertiesOnly: false,
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isValid: true
+        };
+    }
+
+    //Generates a temporary id for add/delete of the import aliases
+    static generateAliasId() {
+        return generateId("sampletype-parent-import-alias-");
+    }
+
+    onFormChange = (evt: any): void => {
+        const {model, updateModel } = this.props;
+        const id = evt.target.id;
+        const value = evt.target.value;
+        updateModel(model.set(getFormNameFromId(id), value) as SampleTypeModel);
+    };
+
+    parentAliasChanges = (id:string, field: string, newValue: any): void => {
+        const {onParentAliasChange} = this.props;
+        onParentAliasChange(id, field, newValue);
+    };
+
+    addParentAlias = (): void => {
+        const {onAddParentAlias} = this.props;
+
+        const newId = SampleTypePropertiesPanelImpl.generateAliasId();
+        const newParentAlias = {
+            id: newId,
+            alias:'',
+            parentValue: undefined,
+            ignoreAliasError: true,
+            ignoreSelectError: true,
+        };
+
+        onAddParentAlias(newId, newParentAlias);
+    };
+
+    renderAddEntityHelper = ():any => {
+        return (
+            <>
+                <span>
+                    {PARENT_ALIAS_HELPER_TEXT}
+                    <p>{helpLinkNode(DERIVE_SAMPLES_ALIAS_TOPIC, "More info")}</p>
+                </span>
+            </>
+        );
+    };
+
+    renderParentAliases = () => {
+        const {model, parentOptions} = this.props;
+        const {parentAliases} = model;
+
+        if (!parentAliases || !parentOptions)
+            return [];
+
+        return parentAliases.valueSeq().map((alias) =>
+            <SampleSetParentAliasRow
+                key={alias.id}
+                id={alias.id}
+                parentAlias={alias}
+                parentOptions={parentOptions}
+                onAliasChange={this.parentAliasChanges}
+                onRemove={this.removeParentAlias}
+            />
+        );
+    };
+
+    removeParentAlias = (index: string): void => {
+        let {onRemoveParentAlias} = this.props;
+        onRemoveParentAlias(index);
+    };
+
+    render = () => {
+        const {model, parentOptions, nameExpressionInfoUrl, nameExpressionPlaceholder, noun} = this.props;
+
+        return (
+            <>
+                <Panel>
+                    <Panel.Body>
```

Comment:
Are we planning to add the <CollapsiblePanelHeader> usage here so that this panel has a header to match the other designers (i.e. assay designer, list designer)?

Reply:
@labkey-ians

Reply:
For consistency sake I think we should, just haven't had time to look at it yet. I don't think it would take much effort, but again haven't really looked at it yet.

Reply:
added